contrib/isa_l/compression/source/compressor/igzip_compressor_impl.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "envoy/compression/compressor/compressor.h"
+
+#include "contrib/isa_l/compression/source/common/base.h"
+#include "isa-l.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Compression {
+namespace Igzip {
+namespace Compressor {
+
+/**
+ * Implementation of compressor's interface.
+ */
+class IgzipCompressorImpl : public Common::Base, public Envoy::Compression::Compressor::Compressor {
+public:
+  IgzipCompressorImpl();
+
+  IgzipCompressorImpl(uint64_t chunk_size);
+
+  // Note that only three compression levels supported in isa-l
+  enum class CompressionLevel : int64_t {
+    Best = 3,
```

Comment:
Is the number not unique?

Reply:
Yeah, they are not unique, like Best = Level3.